pkg/graveler/graveler.go

```diff
@@ -1856,11 +1883,15 @@ func (g *Graveler) deleteUnsafe(ctx context.Context, repository *RepositoryRecor
 	if cachedMetaRangeID != nil && *cachedMetaRangeID != "" {
```

Comment:
I think we can remove the `cachedMetaRangeID`. I don't see it being used and it's also ineffective for compaction, no?

Reply:
It seems like we always pass it empty when calling to `deleteUnsafe`